### PR TITLE
Localized resources update.

### PR DESCRIPTION
**Pull request by fzhu1978**
Newline added at the end of the file.
New strings.json was introduced and mocked.

### CODE REVIEW

**Comment by iwehrman** [comment]
Over to @volfied since he started the review on the original PR.

File: src/nls/de/strings.json [hunk 1]
```diff
@@ -1,5 +1,491 @@
 {
+    "ACTIONS": {
+        "ADD_STROKE": "[8413887] ü_Add Stroke",
```
Comment (volfied) [inline src/nls/de/strings.json]: @fzhu1978 Are these intentionally left untranslated?

Comment (fzhu1978) [inline src/nls/de/strings.json]: Yes, once a new string is added, the process is:
1. ALF scans and harvests.
2. Use localization mock string instead English string, as shown in this line.
3. Send Lockit which contains all untranslated strings to linguists.
4. After translation comes back, ALF use real localized string substitutes the mock string.

**Comment by fzhu1978** [comment]
Oh no, there is a new changes in this pull request that WALF merge from master into feature/localization.

@volfied Can I merge this pull request now?  To keep it clean, I have to create a new request which only contains localization changes.

**Comment by volfied** [comment]
Thanks for the response @fzhu1978. Usually the reviewer merges the Pull Request once all the questions and concerns have been addressed.

In this case, they are, so I'm hitting the green button. Thank you for all your work.

**Comment by fzhu1978** [comment]
Thanks Barkin!

Because ALF will create pull request which contains a lot of unrelated changes, so I am going to create pull request by hand once new translation comes back.

Regards,
Feng

**Pull request merged**